resync/api/routes/admin/backup.py

```
@@ -209,7 +209,7 @@ async def create_database_backup(
         )
         return BackupResponse.from_backup_info(backup)
     except Exception as e:
-        logger.error("create_database_backup_failed", error=str(e))
+        logger.error("create_database_backup_failed", error=str(e))  # type: ignore[call-arg]
```

Ruff (0.15.1)

[warning] Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)